docs/full-node/run-a-full-terra-node/set-up-production.mdx

```diff
@@ -10,13 +10,13 @@ For information about running a validator node, visit the [validator guide](../m
 
 ## Create a dedicated user
 
-Although Terrad does not require a super user account, during the setup process you'll need super user permission to create and modify some files. It is strongly recommended to use a normal user when running Terrad.
+Although Terrad does not require a super user account, during the setup process you'll need super user permissions to create and modify some files. It is strongly recommended to use a normal user when running Terrad.
 
 ## Increase the maximum files Terrad can open
 
-Terrad is set to open 1024 files by default. It is recommended that you increase this amount.
+Terrad can open 1024 files at a time by default. It is recommended that you increase this amount.
```

Comment:
I think this is linux, not Terrad.
Terrad can open a ton of files. I set one node to over 1,000,000 open files allowed.

Reply:
@Jared-TFL Nice catch!  I was focusing on the grammar and didn't catch this one.